lib/login/login.dart

```diff
@@ -0,0 +1,140 @@
+import 'dart:async';
+import 'dart:convert';
+
+import 'package:flutter/material.dart';
+import 'package:off_top_mobile/recordingSession.dart';
+
+import 'package:http/http.dart' as http;
+import 'package:off_top_mobile/components/footer/bottomNavigationTabs.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:google_sign_in/google_sign_in.dart';
+import 'package:modal_progress_hud/modal_progress_hud.dart';
+import 'package:off_top_mobile/login/google_Sign_In.dart';
+
+class LoginPage extends StatefulWidget {
+  const LoginPage({Key key}) : super(key: key);
+
+  @override
+  _LoginPageState createState() => _LoginPageState();
+}
+
+class _LoginPageState extends State<LoginPage> {
+  int userId;
+  String userEmail;
+  String name;
+  bool showLoading = false;
+  UserAuth auth;
+
+  @override
+  void initState() {
+    super.initState();
+    auth = UserAuth();
+    auth.googleSignIn = GoogleSignIn();
+    auth.firebaseAuth = FirebaseAuth.instance;
+  }
+
+  Future<void> makeLoginRequest() async {
```

Comment:
since this method is no longer the actual login functionality, I would rename it to getUserData()